test/jasmine/tests/legend_test.js

```diff
@@ -413,6 +413,15 @@ describe('legend helpers:', function() {
             expect(isReversed({ traceorder: 'reversed' })).toBe(true);
         });
     });
+
+    describe('isHorizontalColumn', function() {
+        var isHorizontalColumn = helpers.isHorizontalColumn;
+
+        it('should return true when option horizontalspacing is "column"', function() {
```

Comment:
Break this into two tests: `should return true when option horizontalspacing is "column"`, and `should return false when option horizontalspacing is "wrapped"`